src/ocm/components/clusterConfiguration/HostInventory.css

```diff
@@ -0,0 +1,3 @@
+.pf-c-content{
+    font-size: var(--pf-global--FontSize--sm) !important;
+}
```

Comment:
Every time the component renders, all elements with this selector will be overridden by this rule.